po/fr.po

```diff
@@ -198,12 +194,11 @@ msgid "Behavior"
 msgstr "Comportement"
 
 #: Settings.ui.h:36
-#, fuzzy
 msgid ""
 "Few customizations meant to integrate the dock with the default GNOME theme. "
 "Alternatively, specific options can be enabled below."
 msgstr ""
-"Assortir le dock avec le thème par défaut. Sinon, vous pouvez personnaliser "
+"Quelques réglages pour intégrer le dock avec le thème par défaut de GNOME. Sinon, vous pouvez personnaliser "
```

Comment:
Since it was "fuzzy", I thought I would change it a bit more, but of course it was fine :). I reverted that change! [d6db0d8114c53a300c37a5e5aac1778d24c077b7]